azure-iot-common/tests/test_asyncio_compat.py

```diff
@@ -0,0 +1,79 @@
+import pytest
+import asyncio
+import sys
+from azure.iot.common import asyncio_compat
+
+pytestmark = pytest.mark.asyncio
+
+
+@pytest.fixture
+def dummy_coroutine():
+    async def coro():
+        return
+
+    return coro
+
+
+@pytest.mark.skipif(sys.version_info < (3, 7), reason="Requires Python 3.7+")
+async def test_get_running_loop(mocker, event_loop):
+    spy_get_running_loop = mocker.spy(asyncio, "get_running_loop")
+    result = asyncio_compat.get_running_loop()
+    assert result == event_loop
+    assert spy_get_running_loop.call_count == 1
+    assert spy_get_running_loop.call_args == mocker.call()
+
+
+@pytest.mark.skipif(sys.version_info >= (3, 7), reason="Requires Python 3.6 or below")
```

Comment:
>sys.version_info >= (3, 7) [](start = 20, length = 26)

did python 3.7 _break_ this? or is it working but we're just not taking this path?